apollo-ast/build.gradle.kts

```diff
@@ -20,12 +20,5 @@ configurations[JavaPlugin.API_CONFIGURATION_NAME].let { apiConfiguration ->
 }
 
 tasks.withType(KotlinCompile::class.java) {
-  kotlinOptions {
```

Comment:
Not completely sure if this is really no longer useful!

Reply:
It's no longuer useful if we merge https://github.com/apollographql/apollo-android/pull/3542

Reply:
Ok, reverting for now :)